package.json

```diff
@@ -1,43 +1,51 @@
 {
-  "name"          : "zipfile",
-  "version"       : "0.5.11",
-  "main"          : "./lib/index.js",
-  "description"   : "C++ library for handling zipfiles in node",
-  "keywords"      : ["zipfile", "uncompress", "unzip", "zlib"],
-  "url"           : "http://github.com/mapbox/node-zipfile",
-  "repository" : {
-           "type" : "git",
-           "url"  : "git://github.com/mapbox/node-zipfile.git"
+  "name": "zipfile",
+  "version": "0.5.11-testciusermapsam",
```

Comment:
dev version?

Reply:
oops, just need to undo this - was testing out the new creds. Thanks!